src/api/app/models/staging/staging_project.rb

```diff
@@ -59,6 +59,17 @@ def overall_state
       @overall_state ||= state
     end
 
+    def assign_managers_group(managers)
+      role = Role.find_by_title!('maintainer')
+      return if relationships.find_by(group: managers, role: role)
```

Comment:
This means the group is already assigned? Then return ``true``, no? Or what does ``Relationship.add_group`` return?

Reply:
As ``Relationship.add_group(self, managers, role, nil, true)`` also returns ``nil`` I'm fine.